bases/br_ibge_pof/caderneta_coletiva_2017/table_config.yaml

```diff
@@ -0,0 +1,467 @@
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id: br_bd_diretorios_brasil
+          table_id: uf
+          column_name: sigla
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: true
+    - name: situacao
+      bigquery_type: string
+      description: Situação do Domicílio
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: id_estrato
+      bigquery_type: string
+      description: Identificador dos estratos do plano amostral da pesquisa
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: id_unidade_primaria_amostragem
+      bigquery_type: string
+      description: Código da Unidade Primária de Amostragem
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: id_domicilio
+      bigquery_type: string
+      description: 'Identificador do domicílio '
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: id_unidade_consumo
+      bigquery_type: string
+      description: Identificador da Unidade de Consumo
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: id_quadro
+      bigquery_type: string
+      description: Número do quadro que o item de despesa/aquisição foi registrado
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: id_codigo_5_bd
+      bigquery_type: string
+      description: Código de 5 dígitos do item
+      temporal_coverage:
+          - 2017-2019
```

Comment:
acho que aqui ficou errado

Reply:
se não for muito trabalho eu colocaria todos os temporal coverage como (1) e o temporal coverage da tabela como 2017,2018. Como são apenas 2 anos e a pesquisa não se repete todo ano me parece fazer mais sentido